src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -19,209 +19,470 @@ If you're new to the topic of running a node, or looking for a less technical pa
+Ethereum clients are able to run on consumer grade computers and don't require any special hardware, like mining machines for example. Therefore, you have various options for deploying the node based on your needs.
 To simplify, let's think about running a node on both a local physical machine and a cloud server:
 
 - Cloud
-  - Providers offer high server uptime, static public IP addresses
+  - Providers offer high server uptime and static public IP addresses
   - Getting dedicated or virtual server can be more comfortable than building your own
   - Trade off is trusting a third party - server provider
-  - Because of required storage size for full node, price of a rented server might get high
+  - Because of the required storage size for full node, the price of a rented server might get high
 - Own hardware
   - More trustless and sovereign approach
   - One time investment
   - An option to buy preconfigured machines
-  - You have to physically prepare, maintain, and potentially troubleshoot the machine
+  - You have to physically prepare, maintain, and potentially troubleshoot the machine and networking
 
 Both options have different advantages summed up above. If you are looking for a cloud solution, in addition to many traditional cloud computing providers, there are also services focused on deploying nodes. For example:
 
-- [QuickNode](https://www.quicknode.com/)
+- [QuikNode](https://www.quiknode.io/)
 - [Blockdaemon](https://blockdaemon.com)
-- [LunaNode](https://www.lunanode.com/)
 - [Alchemy](https://www.alchemy.com/)
 
+Check out also [nodes as a service](/developers/docs/nodes-and-clients/nodes-as-a-service/) for more options on hosted nodes. 
+
 #### Hardware {#hardware}
 
-However, a censorship-resistant, decentralized network should not rely on cloud providers. It's healthier for the ecosystem if you run your own node on hardware. The easiest options are preconfigured machines like:
+However, a censorship-resistant, decentralized network should not rely on cloud providers. It's healthier for the ecosystem if you run your own node on your local hardware. [Estimations](https://www.ethernodes.org/networkType/Hosting) show around 40-60% of nodes is running on cloud which could become a single point of failure. 
 
-- [DappNode](https://dappnode.io/)
-- [Avado](https://ava.do/)
+Depending on your needs and wants, Ethereum clients can run on your computer, laptop, server or single-board computer. Even if it's possible to run clients on your personal computer, having a dedicated machine just for your node can greatly enhance its performance and security while minimizing the impact on your primary computer.
+
+Using your own hardware can be very easy and escalates from simple to more advanced options. Let's look into the requirements and means for running Ethereum clients on your own machine. 
```

Comment:
> escalates from simple to more advanced options.

Not sure what you mean by this?

Reply:
Right, it should say something like 
"spectrum of options (ways of running node) ranges from simple to more advanced ones"